utils/datautils.js

```diff
@@ -49,20 +60,19 @@ function updateClaim(claimId, status) {
 
 function createClaim(user, issueUrl, pullUrl, bounty, status) {
 
-    const claim = {
-        action: 'create',
-        user, issueUrl, pullUrl, bounty, status
-    };
-    fs.writeFile(__dirname + '/../audit/' + new Date().toISOString() + '.json', JSON.stringify(claim), () => {});
 
-    return db.Claim.create({
-        user,
-        issueUrl,
-        pullUrl,
-        repo: pullUrl.split('github.com/')[1].split('/')[1],
-        bounty: bounty,
-        status: status
-    })
+    return github.users.getForUser({
+        username : user
+    }).then(data=>{
```

Comment:
have you tested this as working ? 

aren't you returning the wrong promise form here